addons/mail/static/src/discuss/call/common/device_select.xml

```diff
@@ -2,8 +2,47 @@
 <templates xml:space="preserve">
 
     <t t-name="discuss.CallDeviceSelect">
+        <Dropdown
+            t-if="!useSelect"
+            onStateChanged.bind="openSelect"
+            disabled="!isPermissionGranted(props.kind)"
+             >
+            <button
```

Comment:
The `>` is misindented, should be vertically aligned with `<Dropdown`.
Normally XML sucks and we decide to put all attributes inline exception for `t-att-class` with object syntax, but whatever IMO this is not a big deal here.